examples/data-objects/scribe/src/tools-core/author.ts

```diff
@@ -94,6 +93,15 @@ export interface IScribeMetrics {
 export declare type ScribeMetricsCallback = (metrics: IScribeMetrics) => void;
 export declare type QueueCallback = (metrics: IScribeMetrics, author: IAuthor) => void;
 
+export async function requestSharedString(loader: ILoader, urlBase: string): Promise<ISharedString> {
+    const response = await loader.request({ url: `${urlBase}/sharedstring` });
```

Comment:
I provided similar feedback in some other PR:
Can you please make this code using requestFluidObject by adjusting its signature to not require IFluidRouter, but any object that has request() on it?

Reply:
i don't believe i can use requestFluidObject, as this is a custom request that doesn't return mime type of fluid/object